test/framework/gardener_utils.go

```diff
@@ -576,3 +596,142 @@ func ScaleGardenerScheduler(setupContextTimeout time.Duration, client client.Cli
 func ScaleGardenerControllerManager(setupContextTimeout time.Duration, client client.Client, desiredReplicas *int32) (*int32, error) {
 	return ScaleDeployment(setupContextTimeout, client, desiredReplicas, "gardener-controller-manager", gardencorev1beta1constants.GardenNamespace)
 }
+
+// CreateShoot Creates a shoot from a seed Object and waits until it is successfully reconciled
+func (f *GardenerFramework) CreateSeed(ctx context.Context, seed *gardencorev1beta1.Seed) error {
+	err := retry.UntilTimeout(ctx, 20*time.Second, 5*time.Minute, func(ctx context.Context) (done bool, err error) {
+		_, err = f.createSeedResource(ctx, seed)
+		if apierrors.IsInvalid(err) || apierrors.IsForbidden(err) || apierrors.IsAlreadyExists(err) {
+			return retry.SevereError(err)
+		}
+		if err != nil {
+			f.Logger.Debugf("unable to create seed %s: %s", seed.Name, err.Error())
+			return retry.MinorError(err)
+		}
+		return retry.Ok()
+	})
+	if err != nil {
+		return err
+	}
+
+	// Then we wait for the shoot to be created
+	err = f.WaitForSeedToBeCreated(ctx, seed)
+	if err != nil {
+		return err
+	}
+
+	f.Logger.Infof("Seed %s was created!", seed.Name)
+	return nil
```

Comment:
```suggestion
	return f.WaitForSeedToBeCreated(ctx, seed)
```
If you need the log just put it after this function is called